src/internal/operations/export.go

```diff
@@ -266,10 +264,6 @@ func (op *ExportOperation) do(
 		return nil, clues.Stack(err)
 	}
 
-	opStats.ctrl = op.ec.Wait()
```

Comment:
I don't see this getting moved anywhere.  But also not seeing where we've dropped the wait counter.  Do we no longer need the wait?

Reply:
nope, don't need it. We [were](https://github.com/alcionai/corso/blob/7b6484663b6f8f11e004105eba34dd0dcf60d299/src/internal/m365/export.go#L85-L86) technically telling the status handler that we added 1 collection and then immediately "completing" the one collection we told it about, with an empty status

Since it was returning all zeros anyway and we can't actually get any stats until the collections are consumed by whatever is doing the export (since we don't decide output here) it seemed less foot gun-ish to just remove the thing that could cause stuff to hang